lib/spree/money.rb

```diff
@@ -47,5 +44,30 @@ def format(options = {})
     def ==(other)
       @money == other.money
     end
+
+    private
+
+    def defaults
+      {
+        with_currency: Spree::Config[:display_currency],
+        no_cents: Spree::Config[:hide_cents],
+        decimal_mark: Spree::Config[:currency_decimal_mark],
+        thousands_separator: Spree::Config[:currency_thousands_separator],
+        format: position_to_format(Spree::Config[:currency_symbol_position])
+      }
+    end
+
+    def position_to_format(position)
+      return if position.nil?
+
+      case position.to_sym
+      when :before
+        '%u%n'
+      when :after
+        '%n %u'
+      else
+        raise 'Invalid symbol position'
+      end
+    end
   end
```

Comment:
Nice, I was wondering how to do this without migrating the actual preferences. 💯